docker/README.md

```diff
@@ -41,20 +41,25 @@ Here is an example to serve Resnet50 model using OpenVINO Integration with Tenso
 	
 	To run on default CPU backend:
 
-		$ docker run -t --rm -p 8501:8501 -v <path to Resnet50 model>:/models/resnet -e MODEL_NAME=resnet openvino_tensorflow/tensorflow-serving &
+		$ docker run -t --rm -p 8501:8501 -v <path to Resnet50 model>:/models/resnet -e MODEL_NAME=resnet openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
 
 	To run on iGPU:
 
-		$ docker run -t --rm -p 8501:8501 --device /dev/dri:/dev/dri -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=GPU openvino_tensorflow/tensorflow-serving &
+		$ docker run -t --rm -p 8501:8501 --device /dev/dri:/dev/dri -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=GPU openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
 
 	To run on Myriad:
 
-		$ docker run -t --rm -p 8501:8501 --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=MYRIAD openvino_tensorflow/tensorflow-serving &
+		$ docker run -t --rm -p 8501:8501 --device-cgroup-rule='c 189:* rmw' -v /dev/bus/usb:/dev/bus/usb -v <path to Resnet50 model>:/models/resnet MODEL_NAME=resnet  -e OPENVINO_TF_BACKEND=MYRIAD openvino/openvino_tensorflow_ubuntu20_runtime:2.0.0-serving &
 
 3. Run the script to send inference request from client and get predictions from server.
 
 		$ wget https://raw.githubusercontent.com/tensorflow/serving/master/tensorflow_serving/example/resnet_client.py
 		$ python resnet_client.py
 
+# Prebuilt Images
+
+- [Docker* Hub](https://hub.docker.com/u/openvino/)
+- [Azure* Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/intel_corporation.openvino)
+
 ---
```

Comment:
These links seem to point to OpenVINO images. Will the links be updated to OVTF images after uploading them to DockerHub and Azure Marketplace?

Reply:
These are provisional links, and we'll update them post release. However, OVTF images will be pushed to the same openvino account for DockerHub.

Reply:
That's great. Can we add the link to the specific OVTF image after pushing it? The parent openvino account seems to have many docker images. It may be hard for the users to find the specific image.

Reply:
Specific links for DockerHub are now added. Azure link is subject to change.

Reply:
That's great. Can we also add TF Serving docker image links here? Also, can you change "Prebuilt Images" to "Prebuilt Docker Images"?

Reply:
TF Serving images are present in the same links for ubuntu18_runtime and ubuntu20_runtime with the 2.0.0-serving tag. This will be covered in the README in the respective DockerHub pages.